assets/javascripts/discourse/components/wizard-custom-field.js.es6

```diff
@@ -26,6 +27,18 @@ export default Component.extend(UndoChanges, {
   showAdvanced: alias('field.type'),
   messageUrl: 'https://thepavilion.io/t/2809',
   
+  @discourseComputed('field.type')
+  validations(type) {
```

Comment:
@fzngagan I see what you're thinking here, however I'm not quite convinced yet. Let's discuss.